app/Models/Skill.php

```diff
@@ -1,14 +1,18 @@
 <?php
 
-/**
- * Created by Reliese Model.
- * Date: Thu, 12 Jul 2018 22:39:27 +0000.
- */
-
 namespace App\Models;
 
 use Backpack\CRUD\app\Models\Traits\CrudTrait;
 use Backpack\CRUD\app\Models\Traits\SpatieTranslatable\HasTranslations;
+use Illuminate\Database\Eloquent\Relations\Relation;
+
+Relation::morphMap([
+    'work' => 'App\Models\ExperienceWork',
+    'personal' => 'App\Models\ExperiencePersonal',
+    'education' => 'App\Models\ExperienceEducation',
+    'award' => 'App\Models\ExperienceAward',
+    'community' => 'App\Models\ExperienceCommunity',
+]);
```

Comment:
This should be added to the morphMap in AppServiceProvider.
https://laravel.com/docs/7.x/eloquent-relationships#custom-polymorphic-types